pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# external libraries
+from sympy import Symbol
+from sympy.physics.vector import Point, Dyadic, ReferenceFrame
+from sympy.physics.mechanics import Particle, RigidBody,\
+    dynamicsymbols, outer
+
+# local
+from ..bodies import Body
+
+
+def test_points(point1, point2):
+    assert point1.name == point2.name
+    assert point1._pos_dict == point2._pos_dict
+    assert point1._vel_dict == point2._vel_dict
+    assert point1._acc_dict == point2._acc_dict
```

Comment:
It seems reasonable to add an `__eq__` method to Point for these checks.

But this also begs the question of whether this is really what you need to check in the tests. If you are creating new points inside the body class then testing for this equality seems necessary. But maybe all you have to do is check if two Points produce the same result from `pos_from` or `vel` call.